nucypher/blockchain/eth/agents.py

```diff
@@ -499,8 +500,10 @@ def create_policy(self,
                                                    sender_address=author_address)
         return receipt
 
-    def fetch_policy(self, policy_id: str) -> list:
+    def fetch_policy(self, policy_id: bytes) -> list:
         """Fetch raw stored blockchain data regarding the policy with the given policy ID"""
+        if not len(policy_id) == self._onchain_policy_id_length:
```

Comment:
Why not `!=` :thinking: 

Although, doesn't matter here: should work the same way

Reply:
:heavy_check_mark: